src/app/Mappings/components/AddEditMappingModal.tsx

```diff
@@ -88,7 +89,15 @@ const AddEditMappingModal: React.FunctionComponent<IAddEditMappingModalProps> =
       isOpen
       onClose={onClose}
       actions={[
-        <Button key="confirm" variant="primary" onClick={() => alert('TODO')}>
+        <Button
+          key="confirm"
+          variant="primary"
+          onClick={() => {
+            //TODO: Replace with a real redux call for adding a mapping
+            updateMockStorage({ mappingName, sourceProvider, targetProvider, mappingType });
```

Comment:
We'll need to change this after we merge #25, we can generate the whole mapping object with a helper function I wrote there